src/components/TagFilters.svelte

```diff
@@ -49,4 +66,58 @@
     background: var(--orange-light-1);
     --color: var(--orange);
   }
+
+  :global(body:not(.desktop)) {
+    div {
+      width: 100%;
+      background-color: var(--white);
+      gap: 12px;
+      flex-wrap: nowrap;
+      z-index: 50;
+      position: fixed;
+      top: 73px;
+      left: 0;
+      padding: 24px 20px 16px;
+      overflow-x: auto;
+      -webkit-overflow-scrolling: touch;
+      -ms-overflow-style: -ms-autohiding-scrollbar;
+
+      &::-webkit-scrollbar {
+        display: none;
+      }
+    }
```

Comment:
1. Use `single-line` and `no-scroll` classes
2. Use `position: sticky`

Reply:
I kept single-line because we need to modify flex-wrap not white-space
Also I advice to use position fixed to prevent vertical scroll overlapping filters